modules/core/src/main/java/org/apache/ignite/internal/visor/baseline/VisorBaselineTask.java

```diff
@@ -32,6 +32,7 @@
 import org.apache.ignite.internal.util.typedef.internal.S;
 import org.apache.ignite.internal.visor.VisorJob;
 import org.apache.ignite.internal.visor.VisorOneNodeTask;
+import org.jetbrains.annotations.NotNull;
```

Comment:
We don't use NotNull annotation https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-@Annotations